library/pk_wrap.c

```diff
@@ -1430,20 +1422,21 @@ static int rsa_alt_check_pair(const void *pub, const void *prv,
     size_t sig_len = 0;
     int ret = MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED;
 
-    if (rsa_alt_get_bitlen(prv) != rsa_get_bitlen(pub)) {
+    if (rsa_alt_get_bitlen((mbedtls_pk_context *) prv) !=
```

Comment:
I also noticed that there is a little mix between `const` and non-`const` parameters in `pk_wrap` functions. I would tend to go for the non-`const` solution since this might take some castings away and simplify code readings.
However I will investigate further before taking any direction when I'll have to work on this part

Reply:
Yes, I think we should definitely start with everything non-`const`, get the CI to pass (perhaps even merge), and only then try adding `const` everywhere and see if that works or if we need to be more clever - or perhaps abandon the whole `const` idea.